market_next_door_api/market_next_door_api/tests/views_tests/test_customers.py

```diff
@@ -0,0 +1,99 @@
+from rest_framework.test import APITestCase
+from django.urls import reverse
+from rest_framework import status
+from market_next_door_api.models import *
+import pdb
+
+class CustomerTestCase(APITestCase):
+  def setUp(self):
+    self.customer1 = Customer.objects.create(
+      first_name="George",
+      last_name="Harrison",
+      phone="1234567890",
+      email="george@gmail.com",
+    )
+    self.customer2 = Customer.objects.create(
+      first_name="Janey",
+      last_name="Harrison",
+      phone="1234567890",
+      email="janey@gmail.com",
+    )
+
+  def test_customer_list(self):
+    url =  reverse('customer_list')
```

Comment:
I wonder, does the `reverse()` built-in find the function within the view and run it within the application?